src/HealthChecks.MongoDb/HealthChecks.MongoDb.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MongoDB.Driver" Version="2.28.0" />
+    <PackageReference Include="MongoDB.Driver" Version="[2.28.0,3.0.0)" />

```

Comment:
Should this be 2.30.0?

Reply:
Check the top comment. This is outdated now. We will only support the 3+ version going forward.